shared/src/sources/balancer/info_fetching.rs

```diff
@@ -109,13 +110,16 @@ impl PoolInfoFetching for PoolInfoFetcher {
 
         let tokens = token_data.await?.0;
         let scaling_exponents = self.get_scaling_exponents(&tokens).await?;
+        let amplification_data = amplification_parameter.await?;
+        let amplification_parameter =
+            amplification_data.0.to_big_rational() / amplification_data.2.to_big_rational();
```

Comment:
Nit: Use the constructor with numerator and denominator directly
```suggestion
                let amplification_parameter = BigRational::new(
            amplification_data.0.to_big_int(),
            amplification_data.2.to_big_int(),
        );
```